client/package.json

```diff
@@ -1,42 +1,31 @@
 {
   "name": "client",
-  "version": "0.1.0",
   "private": true,
-  "dependencies": {
-    "@metabase/embedding-sdk-react": "^0.1.22",
-    "@testing-library/jest-dom": "^5.17.0",
-    "@testing-library/react": "^13.4.0",
-    "@testing-library/user-event": "^13.5.0",
-    "react": "^18.3.1",
-    "react-dom": "^18.3.1",
-    "react-scripts": "5.0.1",
-    "web-vitals": "^2.1.4"
-  },
+  "version": "0.0.0",
+  "type": "module",
   "scripts": {
-    "start": "env-cmd -f ../.env react-scripts start",
-    "build": "env-cmd -f ../.env react-scripts build",
-    "test": "env-cmd -f ../.env react-scripts test",
-    "eject": "env-cmd -f ../.env react-scripts eject"
+    "start": "env-cmd -f ../.env vite",
```

Comment:
Compared to the vite default project, I added a start command to keep it compatible with the docs that were using that command.
It's the same as dev, regardless of what the dev is used running, it will work

Reply:
It looks like it only picks up the .env file from the folder where you lunch it, not from the parent folder

Reply:
I was thinking if we can move the "frontend" envs to a separate file, but if it's easier to have a single env (less `cp` commands) we can keep this as-is